static/app/views/performance/transactionSummary/transactionSpans/spanDetails/spanDetailsHeader.tsx

```diff
@@ -0,0 +1,154 @@
+            <SectionBody>
+              {defined(p95ExclusiveTime) ? (
+                <PerformanceDuration abbreviation milliseconds={p95ExclusiveTime} />
+              ) : (
+                '\u2014'
+              )}
+            </SectionBody>
+            <SectionSubtext>{t('p95')}</SectionSubtext>
+          </div>
+          <div data-test-id="section-p99">
+            <SectionBody>
+              {defined(p99ExclusiveTime) ? (
+                <PerformanceDuration abbreviation milliseconds={p99ExclusiveTime} />
+              ) : (
+                '\u2014'
+              )}
+            </SectionBody>
+            <SectionSubtext>{t('p99')}</SectionSubtext>
+          </div>
+        </PercentileHeaderBodyWrapper>
+      </HeaderInfo>
+      <HeaderInfo data-test-id="header-frequency">
+        <StyledSectionHeading>{t('Frequency')}</StyledSectionHeading>
+        <SectionBody>
+          {defined(frequency) && defined(totalCount)
+            ? formatPercentage(Math.min(frequency, totalCount) / totalCount)
+            : '\u2014'}
+        </SectionBody>
+        <SectionSubtext>
+          {defined(avgOccurrences)
+            ? tct('[times] times per event', {times: avgOccurrences.toFixed(2)})
+            : '\u2014'}
+        </SectionSubtext>
+      </HeaderInfo>
+      <HeaderInfo data-test-id="header-total-exclusive-time">
+        <StyledSectionHeading>{t('Total Self Time')}</StyledSectionHeading>
+        <SectionBody>
+          {defined(sumExclusiveTime) ? (
+            <PerformanceDuration abbreviation milliseconds={sumExclusiveTime} />
+          ) : (
+            '\u2014'
+          )}
+        </SectionBody>
+        <SectionSubtext>
+          {defined(frequency)
+            ? tct('[events] events', {events: <Count value={frequency} />})
+            : '\u2014'}
+        </SectionSubtext>
+      </HeaderInfo>
+    </ContentHeader>
+  );
+}
+
+const ContentHeader = styled('div')`
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: ${space(4)};
+  margin-bottom: ${space(2)};
+
+  @media (min-width: ${p => p.theme.breakpoints[1]}) {
+    grid-template-columns: auto repeat(3, max-content);
+    grid-row-gap: 0;
+  }
+`;
+
+const HeaderInfo = styled('div')`
+  ${overflowEllipsis};
+  height: 78px;
+`;
+
+const StyledSectionHeading = styled(SectionHeading)`
+  margin: 0;
+`;
+
+const SectionBody = styled('div')<{overflowEllipsis?: boolean}>`
+  font-size: ${p => p.theme.fontSizeExtraLarge};
+  padding: ${space(0.5)} 0;
+  max-height: 32px;
+`;
+
+const SectionSubtext = styled('div')`
+  color: ${p => p.theme.subText};
+  font-size: ${p => p.theme.fontSizeMedium};
+`;
+
+const PercentileHeaderBodyWrapper = styled('div')`
+  display: grid;
+  grid-template-columns: repeat(3, max-content);
+  gap: ${space(3)};
+`;
+
+export const SpanLabelContainer = styled('div')`
+  ${overflowEllipsis};
+`;
+
+const EmptyValueContainer = styled('span')`
+  color: ${p => p.theme.gray300};
+`;
```

Comment:
This stuff didn't really change right?

Reply:
nope, I just moved the header to its file